demo/admin/src/pages/PageContentBlock.tsx

```diff
@@ -20,4 +40,73 @@ export const PageContentBlock = createBlocksBlock({
         linkList: LinkListBlock,
         fullWidthImage: FullWidthImageBlock,
     },
+    additionalFields: {
+        userGroup: {
+            defaultValue: "All",
+        },
+    },
+    AdditionalContextMenuItems: ({ block, onChange, onMenuClose }) => {
```

Comment:
what is `block`, the block state?

Reply:
It's a single child of the blocks block. Internally we call this `blocksBlockItem`. I may change it here to `item`, but that may be confusing with `AdditionalContextMenuItems`. Maybe `blocksBlockItem`?

Reply:
> Maybe blocksBlockItem?

:+1: